src/App.tsx

```diff
@@ -1,9 +1,19 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+
+  function searchInput(item:string) {
+    return item.toLowerCase().includes(query.toLowerCase().trim());
+  }
+
+  const visibleMovies
+  = moviesFromServer.filter(movie => searchInput(movie.title)
+  || searchInput(movie.description));
```

Comment:
If you feel fancy, you can separate words into array and search if every word is included.